sinks/service.go

```diff
@@ -11,31 +11,39 @@ package sinks
 import (
 	"context"
 	"github.com/mainflux/mainflux"
+	mfsdk "github.com/mainflux/mainflux/pkg/sdk/go"
 	"github.com/ns1labs/orb/pkg/errors"
+	"github.com/ns1labs/orb/pkg/types"
+	"github.com/ns1labs/orb/sinks/backend/prometheus"
+	"go.uber.org/zap"
 	"time"
 )
 
-var (
-	ErrCreateSink = errors.New("failed to create sink")
-)
-
 type Service interface {
-	// CreateSink creates new data sink
-	CreateSink(ctx context.Context, token string, s Sink) (Sink, error)
+	SinkService
```

Comment:
I think this wrapper probably isn't necessary, and we should just use SinkService instead of Service unless you can think of a use case?